pennylane/templates/subroutines/qpe.py

```diff
@@ -200,6 +200,30 @@ def estimation_wires(self):
         """The estimation wires of the QPE"""
         return self._hyperparameters["estimation_wires"]
 
+    @property
+    def error(self):
```

Comment:
`spectral_norm_error` looks good to me.

Reply:
This is the error method inherited from the ErrorOperation class. It should not be renamed.

Reply:
Decided to keep it as `error` because the docstring is clear and it is a required `abstractmethod`.
Changes to support other types of errors can be supported by adding a keyword argument like `QPE.error(calculation='SpectralNorm')`